geometry_msgs/Point position
float32 radius 
---
uint32[] object_ids